### PR TITLE
Fix some typo

### PR DESCRIPTION
**Pull request by xianlubird**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by dixudx** [comment]
/release-note-none

@xianlubird  Could you please help find other typos in `staging/src/k8s.io/apiserver/`? I prefer to fix them together in a single PR. Thanks.

File: staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go [hunk 1]
```diff
@@ -239,7 +239,7 @@ func (f *fancyResponseWriterDelegator) Hijack() (net.Conn, *bufio.ReadWriter, er
 	// fake a response status before protocol switch happens
 	f.processCode(http.StatusSwitchingProtocols)
 
-	// This will be ignored if WriteHeader() function has aready been called.
+	// This will be ignored if WriteHeader() function has already been called.
 	// It's not guaranteed Audit-ID http header is sent for all requests.
 	// For example, when user run "kubectl exec", apiserver uses a proxy handler
```
Comment (nikhita) [inline staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go]: when _a_ user _runs_...

**Comment by nikhita** [comment]
@xianlubird Typo fixes don't need a release note. Can you remove the release note from the PR body? Thanks!

**Comment by xianlubird** [comment]
@nikhita  Done

File: staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go (hunk 1 above)
Comment (dixudx) [reply staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go]: @xianlubird  This is still not addressed.

Comment (xianlubird) [reply staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go]: @dixudx @nikhita Done

**Comment by xianlubird** [comment]
ping @dixudx

File: staging/src/k8s.io/apiserver/pkg/authorization/union/union.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ limitations under the License.
 // subauthorizer returns a NoOpinion, then the union authorizer moves onto the
 // next authorizer or, if the subauthorizer was the last authorizer, returns
 // NoOpinion as the aggregate decision. I.e. union authorizer creates an
-// aggregate decision and supports short-circut allows and denies from
+// aggregate decision and supports short-circuit allows and denies from
```
Comment (nikhita) [inline staging/src/k8s.io/apiserver/pkg/authorization/union/union.go]: All these typos have been fixed in https://github.com/kubernetes/kubernetes/pull/59464. Closing.

/close

**Pull request closed without merging**